src/scipp/html/formatting_html.py

```diff
@@ -220,6 +203,23 @@ def summarize_attrs(attrs, embedded_in=None):
     return f"<ul class='sc-var-list'>{attrs_li}</ul>"
 
 
+def find_bin_edges(var, ds):
+    """
+    Checks if the coordinate contains bin-edges.
+    """
+    bin_edges = []
+    for idx, dim in enumerate(var.dims):
+        length = var.shape[idx]
+        if not ds.dims:
+            # Have a scalar slice.
+            # Cannot match dims, just assume length 2 attributes are bin-edge
+            if length == 2:
+                bin_edges.append(dim)
+        elif dim in ds.dims and ds.shape[ds.dims.index(dim)] + 1 == length:
+            bin_edges.append(dim)
+    return bin_edges
```

Comment:
Can this now use the newly bound C++ method?

Reply:
This would not be any less complex. It would have to check both `meta` and `masks`, except that `Dataset` does not have `masks`. So there needs to be quite a lot of logic.